helm/alfresco-content-services/templates/deployment-filestore.yaml

```diff
@@ -76,13 +76,11 @@ spec:
             periodSeconds: {{ .Values.filestore.livenessProbe.periodSeconds }}
             failureThreshold: 1
             timeoutSeconds: {{ .Values.filestore.livenessProbe.timeoutSeconds }}
-          {{- if .Values.persistence.filestore.enabled }}
           volumeMounts:
           - name: data
             mountPath: {{ .Values.persistence.filestore.data.mountPath }}
             subPath: {{ .Values.persistence.filestore.data.subPath }}
-          {{- end }}
-      {{- if .Values.persistence.filestore.enabled }}
+      {{- if and .Values.persistence.filestore.enabled .Values.persistence.dynamicProvisioning }}
```

Comment:
I thought chown didn't work when using dynamic provisioning. Shouldn't that if statement work the other way round?

Reply:
Oh. It might be unfortunate parameter name on my part - chown doesn't work with dynamic provisioning as you noticed (yet, but it might one day), so it'd be probably better to rename it to something like `chownWithDynamicProvisioning`